java/examples/src/java/org/apache/orc/examples/Driver.java

```diff
@@ -24,11 +24,47 @@
 import org.apache.commons.cli.ParseException;
 import org.apache.hadoop.conf.Configuration;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Driver program for the java ORC examples.
  */
 public class Driver {
 
+  private static final Map<String, ExampleInfo> exampleMaps = new HashMap<>();
+
+  static {
+    registerExample("write",
+            new ExampleInfo("write a sample ORC file", CoreReader.class));
+    registerExample("read",
+            new ExampleInfo("read a sample ORC file", CoreWriter.class));
+    registerExample("write2",
+            new ExampleInfo("write a sample ORC file with a map", AdvancedWriter.class));
+    registerExample("read2",
+            new ExampleInfo("read a sample ORC file with a map", AdvancedReader.class));
+    registerExample("compressWriter",
+            new ExampleInfo("write a ORC file with snappy compression", CompressionWriter.class));
+    registerExample("inMemoryEncryptionWriter",
+            new ExampleInfo("write a ORC file with encryption", InMemoryEncryptionWriter.class));
+    registerExample("inMemoryEncryptionReader",
+            new ExampleInfo("read a ORC file with encryption", InMemoryEncryptionReader.class));
+  }
```

Comment:
Please don't refactor like this in this PR. We can consider this independently.

Reply:
OK, But in that case, we need to use more if else here.

Reply:
I will remove it.